strings/redpill.json

```diff
@@ -0,0 +1,24 @@
+{
+	"redpill_questions": [
+		"What happened to the ceiling?",
+		"Why is it called the emergency shuttle if it arrives every single shift?",
```

Comment:
This is ported from tg where they only have the emergency shuttle and no crew transfer
Its just not that the shuttle is the same. But tg always end up evuacating.
While para doesnt

Reply:
Crew transfer still uses the emergency shuttle, question is valid